src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -95,6 +94,11 @@ trait Backend {
    */
   def stdoutStderr(descriptor: WorkflowDescriptor, callName: String, index: ExecutionIndex): CallLogs
 
+  /**
+   * Provides a function that given a WdlFile, returns its hash.
```

Comment:
Hum why ? a `FileHasher` is a `WdlFile => SymbolHash`, and this function returns one of those. Maybe the comment should say "Returns a function that given a WdlFile, returns its hash".